apiary.apib

```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
 
 + Response 401
 
-    [1167][]
\ No newline at end of file
+    [1167][]
+ 
+## Create Competitor Enterprise [/v1/signup/competitor/{businessNumber}{?api_key}]
+New Competitor can be created:
+
++ Parameters
+
+    + businessNumber (string, required) ... the business ID
+    + api_key (string, required) ... the API key
+
+### Create New Competitor Enterprise [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+    
+    curl -H "content-type: application/json" -H "accept:application/json" -X POST -d '{"businessName": "smb_comp","userEmailId": "rhonda@reviewmanagement.com"}' "https://api.birdeye.com/resources/v1/signup/competitor/151134205012586?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451" -v
+
+<center><b>Example Payload</b></center>
+
+{
+    "businessName": "CompE",
+    "userEmailId": "rhonda@reviewmanagement.com"
+}
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Request Payload</caption>
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>businessName</td>
+            <td>Name of the Competitor Enterprise that needs to be added</td>
+        </tr>
+        <tr>
+            <td>userEmailId</td>
+            <td>User Email</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+#### Response
+
+<center><b>Example Response</b></center>
+
+{
+    "competitorId": 100037739,
+    "templateid": 0,
+    "isAggregationRequired": 1,
+    "type": "Enterprise-Location-Competitor",
+    "businessNumber": 161295237142347,
+    "businessAlreadyPresent": false,
+    "sfdcAccountId": null,
```

Comment:
@khyati-gupta Why does client care of sfdcAccountid and sfdcOpportunityId? these are internal fields for Birdeye. Why do we have them in the response?